### PR TITLE
Feature/logging fixes testing

### PR DESCRIPTION
**Pull request by Andredance**
1. Fix for softmax. (able to convert different axis now).
2. Fix for instancenorm.  TF uses TFA (tensorflow-addons) instead of tf.contrib.
3. New testing with pytest.
4. Few additional tests of models from torchvision.
5. New logging hierarchy for better configuration from outside lib.

Some of tests can't be run on CPU only (they are skipped by pytest). Tested with torch-1.5.0 and torch-1.4.0 + tf-2.3.1.

### CODE REVIEW

**Comment by nerox8664** [comment]
Hello, @Andredance. Thanks for your time. I accept the pull-request.
